_docs/tasks/traffic-management/ingress.md

```diff
@@ -237,7 +237,7 @@ The following are known limitations of Istio Ingress:
      169.47.243.100
      ```
 
-     along with the istio-ingress service's nodePort for port 80:
+     along with the istio-ingress service's nodePort for port 443:
```

Comment:
that seems odd, what caused this change ? the ingress will only listen on 443 if you have certs

Reply:
nvm I didn't see this was in the https ingress section

Reply:
(We chatted in slack but posting here in case others are curious.)

This is updating the HTTPS section of the doc. The ingress doc covers HTTP and HTTPS, and we'd previously directly copy/pasted the HTTP verification section into the HTTPS section; this corrects the HTTPS section to use 443 and https rather than http in the curl URL.